src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -98,37 +98,38 @@
  * <li>be associated with a specific binary application,
  * or a class of binary applications.</li>
  * </ul>
- *
+ * <p>
  * When such approvals need to given interactively, it may impede the normal
  * operation of the application until approved, and if approval is denied
  * or not possible, or cannot be made persistent then it will degrade
  * the functionality of this class and in turn any part of the operation
  * of the application which is dependent on it.
  *
- * @author      Robi Khan
- * @since       1.3
+ * @author Robi Khan
+ * @since 1.3
  */
 public class Robot {
     private static final int MAX_DELAY = 60000;
     private RobotPeer peer;
     private boolean isAutoWaitForIdle = false;
     private int autoDelay = 0;
     private static int LEGAL_BUTTON_MASK = 0;
+    private static Rectangle[] allScreenBounds;
 
     private DirectColorModel screenCapCM = null;
 
     /**
      * Constructs a Robot object in the coordinate system of the primary screen.
      *
-     * @throws  AWTException if the platform configuration does not allow
-     * low-level input control.  This exception is always thrown when
-     * GraphicsEnvironment.isHeadless() returns true
-     * @see     java.awt.GraphicsEnvironment#isHeadless
+     * @throws AWTException if the platform configuration does not allow
+     *                      low-level input control.  This exception is always thrown when
+     *                      GraphicsEnvironment.isHeadless() returns true
+     * @see java.awt.GraphicsEnvironment#isHeadless
      */
     public Robot() throws AWTException {
         checkHeadless();
         init(GraphicsEnvironment.getLocalGraphicsEnvironment()
-            .getDefaultScreenDevice());
+                .getDefaultScreenDevice());
     }
```

Comment:
Nothing's changed here… These look as unrelated to the problem that's being addressed.